build/ssh_banner.txt

```diff
@@ -0,0 +1,24 @@
+                 _                    _   _                    
+  _ __    __ _  | |__  ___     __ _  (_) | |_     
+ | '  \  / _` | | / / / -_)   / _` | | | |  _|   
+ |_|_|_| \__,_| |_\_\ \___|   \__, | |_|  \__|     
+                              |___/              
+                   ___                                   ___           ___     
+    _____         /  /\          ___         ___        /  /\         /  /\    
+   /  /::\       /  /:/_        /  /\       /  /\      /  /:/_       /  /::\   
+  /  /:/\:\     /  /:/ /\      /  /:/      /  /:/     /  /:/ /\     /  /:/\:\  
+ /  /:/~/::\   /  /:/ /:/_    /  /:/      /  /:/     /  /:/ /:/_   /  /:/~/:/  
+/__/:/ /:/\:| /__/:/ /:/ /\  /  /::\     /  /::\    /__/:/ /:/ /\ /__/:/ /:/___
+\  \:\/:/~/:/ \  \:\/:/ /:/ /__/:/\:\   /__/:/\:\   \  \:\/:/ /:/ \  \:\/:::::/
+ \  \::/ /:/   \  \::/ /:/  \__\/  \:\  \__\/  \:\   \  \::/ /:/   \  \::/~~~~ 
+  \  \:\/:/     \  \:\/:/        \  \:\      \  \:\   \  \:\/:/     \  \:\     
+   \  \::/       \  \::/          \__\/       \__\/    \  \::/       \  \:\    
+    \__\/         \__\/                                 \__\/         \__\/    
+
+
+	   A CTF challenge by Shay Nehmad 
+	   Visit https://mrnice.dev
+	   
+This is a game server. Please try to not mess it up ¯\_(ツ)_/¯
+If you find any issues, let me know @ShayNehmad on Twitter.
+
```

Comment:
Maybe add some notes about the server config?
"Your shell is ZSH with oh-my-zsh etc."

Reply:
Will be on the site. No need to see each login